src/core/Table/filters/tableFilters.tsx

```diff
@@ -49,8 +49,13 @@ export const tableFilters = {
   ) => <TextFilter {...props} translatedLabels={translatedLabels} />,
   /**
    * Date range filter.
-   * By default it handles `en-us` date format. If you want to use other format
+   *
```

Comment:
why empty line?

Reply:
I thought you will ask this 😁
Just to have nicer view/better readability:
![image](https://user-images.githubusercontent.com/36186912/123097603-f6df0480-d438-11eb-809c-9ec6fa6b5d6b.png)